### ISSUE TITLE
import:backendusers doesn't import anything

### ISSUE THREAD

**Issue by DenisMir** [MaxServ/t3ext-yaml-configuration#6]
I have exported my base backendusers to a yml file via `export:backendusers` which worked quite fine. When trying to import these backend users via `import:backendusers` from the exported yml file I'm getting only the following output: 

![bildschirmfoto 2016-03-09 um 15 17 45](https://cloud.githubusercontent.com/assets/157281/13638018/9a545c96-e60a-11e5-9ed2-81d45797f0f9.png)

When looking into the `be_users` table nothing happened. No users got inserted from the yml file. Does the import process only update existing users or what could be wrong here? 

The same happens to me when trying to import some exported basic pages to the `pages` table with the following command: 

`./typo3cms import:table --table=pages --file=/vagrant/pages.yml --match-fields=title` 

The yml file contains the following data: 

``` yml
TYPO3:
    Data: { pages: [{ uid: '1', sorting: '256', perms_userid: '1', perms_user: '31', perms_group: '27', cruser_id: '1', title: Website, doktype: '4', urltype: '1', shortcut_mode: '1' }, { uid: '2', pid: '1', sorting: '256', perms_userid: '1', perms_user: '31', perms_group: '27', cruser_id: '1', title: Home, doktype: '1', urltype: '1' }] }
```

After executing the command the `pages` table is still empty. 

So to sum up. Exporting works perfectly and importing doesn't work at all.

My setup: TYPO3 7.6.4 LTS in `COMPOSER_MODE`

**-- Update --**

I have found the root cause of the issue. 

https://github.com/MaxServ/t3ext-yaml-configuration/blob/master/Classes/Command/ImportCommandController.php#L128

The usage of `GeneralUtility::getFileAbsFileName` forces me to put the import files under the webroot of the project. Otherwise absolute path names like `/vagrant/pages.yml` lead to an empty return value which cause the import process to fail.

**Comment by DenisMir** [comment]
Fixed with #7
